roles/example-cnf-destroy/tasks/machine-fix-sub.yaml

```diff
@@ -19,7 +19,7 @@
   loop: "{{ nodes.resources }}"
 - name: delete machine if associated node is not found
   k8s:
-    api_version: machine.openshift.io/v1beta1
+    api_version: machine.openshift.io/v1
```

Comment:
Would this affect example-cnf when running in OCP < 4.12 ?

Reply:
for the moment I'm not even sure that it's a proper fix for 4.12 :) we'll see in one hour

Reply:
I tested on 4.7, 4.10, 4.11, and 4.12 - the stable API version /v1 does not harm. Unfortunately, it does not the original issue either, [here is the API we do need to fix](https://github.com/rh-nfv-int/trex-container-app/pull/2).

Reply:
I do not see information about the deprecation of this API: https://docs.openshift.com/container-platform/4.11/rest_api/machine_apis/machineset-machine-openshift-io-v1beta1.html

Reply:
@betoredhat, the group first appears as v1alpha1 and is then promoted to v1beta1 and finally graduates to v1:

- v1alpha1 is usually disabled by default.
- v1beta1 is enabled by default but may change in incompatible ways in the next release.
- v1 is GA and is supposed to be stable for many releases.

The idea is to use v1 whenever possible, to ensure stability.